_posts/2018-12-03-commons-clause.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Commons Clause at Zalando
+date: 2018-12-03
+author: The Open Source Team
+---
+
+# Introduction
+- What is the CC license?
+- Where/why was it created?
+
+# The Spirit of Open Source
+- The importance of Freedom 0
+- When you commit to Open Source, you commit to your community
+- Commons Clause breaks this covenant
+
+# The Problems of Monetisation
+- Can't sell something that is Free/free
+- So what can we sell?
+  - Consulting
+  - Feature development
+  - Training
+  - Lots of things!
+- It can be done! Look at RedHat!
+
+# Taking A Stand Against Commons Clause
```

Comment:
Redis Labs Modules already forked: https://www.zdnet.com/article/redis-labs-and-common-clause-attacked-where-it-hurts-with-open-source-code/